docs/testing/01-overview.md

```diff
@@ -25,136 +25,14 @@ choose when to use each type of testing:
-import { Config } from '@stencil/core';
-
-export const config: Config = {
-  testing: {
-    testPathIgnorePatterns: [...]
-  }
-};
-```
-
-### Command Line Arguments
-
-While the Stencil CLI offers a certain set of command line flags to specify e.g. which types of tests to run, you also have access to all Jest options through the CLI. For example to specify a single test, you can pass in a positional argument to Jest by adding a `--`, e.g.:
-
-```sh
-# run a single unit test
-npx stencil test --spec -- src/components/my-component/my-component.spec.ts
-# run a single e2e test
-npx stencil test --e2e -- src/components/my-component/my-component.e2e.ts
-```
-
-Next to positional arguments, Stencil also passes along [certain](https://github.com/ionic-team/stencil/blob/54d4ee252768e1d225baababee0093fdb0562b83/src/cli/config-flags.ts#L38-L85) Jest specific flags, e.g.:
-
-```sh
-# enable code coverage
-npx stencil test --spec --coverage
-```
-
-You can find more information about [Jest CLI options](https://jestjs.io/docs/cli) in the project documentation.
-
-## Running and Debugging Tests in VS Code
-
-Adding the following configurations to `.vscode/launch.json` will allow you to use the VS Code Debugger to run the Stencil test runner for the currently active file in your editor.
-
-To use the below configuration:
-1. Ensure the test file you want to run is open and in the current active window in VS Code.
-2. Select the debug configuration to run:
-   1. 'E2E Test Current File' will run the end-to-end tests in the active test file
-   2. 'Spec Test Current File' will run the spec tests in the active test file
-3. Hit the play button to start the test.
-
-```json title=".vscode/launch.json"
-{
-  "configurations": [
-    {
-      "type": "node",
-      "request": "launch",
-      "name": "E2E Test Current File",
-      "cwd": "${workspaceFolder}",
-      "program": "${workspaceFolder}/node_modules/.bin/stencil",
-      "args": ["test", "--e2e", "--", "--maxWorkers=0", "${fileBasename}"],
-      "console": "integratedTerminal",
-      "internalConsoleOptions": "neverOpen"
-    },
-    {
-      "type": "node",
-      "request": "launch",
-      "name": "Spec Test Current File",
-      "cwd": "${workspaceFolder}",
-      "program": "${workspaceFolder}/node_modules/.bin/stencil",
-      "args": ["test", "--spec", "--", "${fileBasename}"],
-      "console": "integratedTerminal",
-      "internalConsoleOptions": "neverOpen"
-    }
-  ]
-}
-```
+Stencil currently supports the following tools for testing components:
 
-:::tip
-Windows users: The `program` value should be set to `"${workspaceFolder}/node_modules/bin/stencil"`.
-If that value does not work, please try`"${workspaceFolder}/node_modules/@stencil/core/bin/stencil"`.
-:::
+- [__Stencil Test Runner__](./stencil-testrunner/01-overview.md): a built-in test runner based on Jest for unit and end-to-end testing with Puppeteer to run within an actual browser in order to provide more realistic results.
```

Comment:
This link doesn't look obviously like a link on the site:

<img width="579" alt="Screenshot 2024-02-07 at 1 27 45 PM" src="https://github.com/ionic-team/stencil-site/assets/6207644/8422b186-edf6-4d98-b19f-1b75291ec79b">

without the bold its "linky-ness" is visually obvious:

<img width="447" alt="Screenshot 2024-02-07 at 1 28 22 PM" src="https://github.com/ionic-team/stencil-site/assets/6207644/9f9fdf1e-a92a-4ccb-8557-fa506623b2f3">

Might be a styling issue, but I think unless we're empowered to change the theme we might want to do something like this:

```suggestion
- [Stencil Test Runner](./stencil-testrunner/01-overview.md): a built-in test runner based on Jest for unit and end-to-end testing with Puppeteer to run within an actual browser in order to provide more realistic results.
```

Reply:
Yeah, this looks way better. Thanks!